pkg/client/interface.go

```diff
@@ -274,4 +274,18 @@ type SessionQueryClient interface {
 		serviceId string,
 		blockHeight int64,
 	) (*sessiontypes.Session, error)
+	// GetParams queries the chain for the current session module parameters.
+	GetParams(ctx context.Context) (*sessiontypes.Params, error)
+	// GetSessionGracePeriodEndHeight returns the height at which the grace period for
+	// the session ending with sessionEndHeight elapses.
+	GetSessionGracePeriodEndHeight(ctx context.Context, sessionEndHeight int64) (int64, error)
+	// GetSessionGracePeriodBlockCount returns the number of blocks in the grace period
+	// for the session which includes queryHeight.
+	GetSessionGracePeriodBlockCount(ctx context.Context, queryHeight int64) (uint64, error)
+	// IsWithinGracePeriod returns true if the grace period for the session ending with
```

Comment:
Where is this being use?

Reply:
![image](https://github.com/pokt-network/poktroll/assets/600733/a8425242-493c-4332-9ee3-11c42609f20f)